docs/team/jingyenloh.md

```diff
@@ -3,44 +3,46 @@ layout: page
 title: Loh Jing Yen's Project Portfolio Page
 ---
 
-## Project: AddressBook Level 3
+## Project: ResiReg
 
-AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+**ResiReg** is a desktop residential college management application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 20 kLoC.
```

Comment:
I don't think the purpose of the application is quite right as the application is supposed to be used to manage a residential college and not teach Software Engineering principles, which is the purpose of AB3. I think that there might have been an oversight regarding this.

Reply:
Thanks for this! I've remedied it in the latest commit.